.github/scripts/import_experiment.py

```diff
@@ -52,18 +54,28 @@ def generate_ftl_file(recipe):
                 comment = match.value.get("comment", "")
                 if id in parsed_ids:
                     # We already parsed a string with this ID. Verify that the
-                    # text is the same. If not, print a warning.
+                    # text is the same. If not, store a warning.
                     if parsed_ids[id] != text:
-                        print(f"WARNING: the ID {id} is used with different values")
-                        print(f"Previous: {parsed_ids['id']}")
-                        print(f"Current: {text}")
+                        warnings.append(
+                            "\n---\n\n"
+                            f"The string with ID `{id}` is defined with different values throughout the recipe.\n"
+                            f"Previous value: `{parsed_ids[id]}`\n"
+                            f"New value: `{text}`\n"
```

Comment:
`Initial value` makes sense, since it's the first one we read. For the second, maybe `Different value (ignored)`? It also makes it clear that the 2nd value has been ignored.

Reply:
Yeah that sounds good. Also works if there are for example 3 instances of the same ID.